### DISCUSSION TITLE
How to preserve creation time stamp after modification

### DISCUSSION THREAD

**Discussion by jkilopu** [helix-editor/helix#11011] [Q&A] [answered]
Hey, folks! I’m editing notes with Helix on MacOS. Whenever I open a file, make changes, and save it using `:write`, the creation timestamp gets updated. Since I use this timestamp to organize my notes, I’d like to keep it unchanged.
I did some Googling and found that Vim has the same default behavior but offers a workaround: [OSX - How to get the creation & modification time of a file from the command line](https://stackoverflow.com/questions/34123076/osx-how-to-get-the-creation-modification-time-of-a-file-from-the-command-lin).
Does anyone know how to do this in Helix?

**Comment by pascalkuthe** [comment, marked as answer]
our file writing works the same as vims now so this doens't work with helix for the same versions. Helix is not extensible enough to implement such an options as a plugin/config. I am not opposed to add a special path to macos that would also copy over birthtime if it can be done in a reasonable amount of complexity and perferably without additional dependencies if somebody creates a pr for it.
